src/main/java/nl/knaw/dans/filemigration/core/EasyFileLoader.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2021 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.filemigration.core;
+
+import io.dropwizard.hibernate.UnitOfWork;
+import nl.knaw.dans.filemigration.db.EasyFileDAO;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Iterator;
+import java.util.Spliterator;
+import java.util.Spliterators;
+import java.util.stream.StreamSupport;
+
+public class EasyFileLoader {
+  private static final Logger log = LoggerFactory.getLogger(EasyFileLoader.class);
+
+  private final EasyFileDAO dao;
+
+  public EasyFileLoader(EasyFileDAO dao) {
+    this.dao = dao;
+  }
+
+  public void loadFromDatasetIds(Iterator<String> ids) {
+    Spliterator<String> spliterator = Spliterators.spliteratorUnknownSize(ids, Spliterator.ORDERED);
+    StreamSupport.stream(spliterator, false).forEach(this::loadFromDatasetId);
+  }
+
+  @UnitOfWork
+  public void loadFromDatasetId(String id) {
+    for (Object ef : dao.findByDatasetId("easy-dataset:9")) {
+      log.info("ef = {}" , ef);
```

Comment:
I'm clearly blind for something. `trace` does not work despite 
https://github.com/DANS-KNAW/dd-verify-file-migration/blob/752502403c8fd0407cd8ebe0875718791a5ab176/src/test/resources/debug-etc/config.yml#L17-L22

also tried the following in `etc/config.yml` (which otherwise equals the file above except for the DB password) what did produce the executed query but `log.isTracelLevel()` stubbornly returns false here.
```
  loggers:
    "io.dropwizard": INFO
    "org.hibernate.SQL": DEBUG
    "dans.knaw.nl": TRACE
```

Reply:
Solved in https://github.com/DANS-KNAW/dans-module-archetype/commit/13335d609f9f20a0b627cbd396a0a3280f78a4e4